lib/src/booleans.dart

```diff
@@ -5,9 +5,11 @@ import 'bool_extensions.dart';
 /// Use [BoolExtensions] instead
 ///
 /// `since 0.0.1`
+@Deprecated('Use [BoolExtensions.isFalse]')
```

Comment:
I'm not sure about making it Deprecated since it is useful as a static method reference (e.g. to be point-free)

Reply:
Yep, agreed.